src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1181,10 +1186,14 @@ void RaftReplDev::on_log_found(logstore_seq_num_t lsn, log_buffer buf, void* ctx
     }
 
```

Comment:
Not directly related to this PR, but could impact some of the assert in underlying layer, is SM long run currently use debug build or release build?

Reply:
Debug build